test/get-selenium-hub-test.js

```diff
@@ -29,177 +26,124 @@ describe('getRunningProcessType', () => {
 });
 
 describe('getSeleniumStatusUrl', () => {
-  // selenium 4 version
-  const seleniumVersion = '4.5.0';
+  /** @type {ReadonlyArray<{ version: '3.141.59' | '4.5.0', args: string[],  expected: string}>} */
```

Comment:
**FYI**: 

* The `version` is required and renamed from `seleniumVersion` 
* The `expectedUrl` is renamed to as `expected`
* Inlines the URL paths
* Removes the test cases that show the `--grid-url` arg is ignored. It is unnecessary.